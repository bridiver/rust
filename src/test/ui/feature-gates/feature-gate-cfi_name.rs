// compile-flags: --crate-type=rlib

#[repr(transparent)]
#[cfi_name = "h"] //~ERROR the `#[cfi_name]` attribute is an experimental feature
pub struct UnsignedChar(u8);
