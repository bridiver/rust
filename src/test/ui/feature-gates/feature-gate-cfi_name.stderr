error[E0658]: the `#[cfi_name]` attribute is an experimental feature
  --> $DIR/feature-gate-cfi_name.rs:4:1
   |
LL | #[cfi_name = "h"]
   | ^^^^^^^^^^^^^^^^^
   |
   = note: see issue #89653 <https://github.com/rust-lang/rust/issues/89653> for more information
   = help: add `#![feature(cfi_name)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
